docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -784,43 +784,121 @@ The output of the query has the following columns:
       * ``EQUALITY_DELETES(2)``
   * - ``file_path``
     - ``varchar``
-    - The data file location
+    - The data file location.
```

Comment:
nit: The other descriptions in `iceberg.rst` are ending without `.`

Reply:
@findinpath I think I added in the `files` table doc as well, and looks like was reviewed by @mosabua https://github.com/trinodb/trino/pull/11206#pullrequestreview-904846629
but I can remove the `.`

Reply:
@findepi done with this change.